sopel/modules/wikipedia.py

```diff
@@ -109,6 +111,26 @@ def configure(config):
     )
 
 
+def choose_lang(bot, trigger):
+    """Determine what language to use for queries based on sender/context."""
+    user_lang = bot.db.get_nick_value(trigger.nick, 'wikipedia_lang')
+    channel_lang = None if trigger.sender.is_nick() else bot.db.get_channel_value(trigger.sender, 'wikipedia_lang')
```

Comment:
Instead of using `if/elif/else` structure, I'd use a `if then return` structure:

```
if user_lang:
    return user_lang

if not trigger.sender.is_nick():
    channel_lang = bot.db.get_channel_value(trigger.sender, 'wikipedia_lang')
    if channel_lang is not None:
        return channel_lang

if bot.config.wikipedia.lang_per_channel:
    # [...]
    return lang

return bot.config.wikipedia.default_lang
```

Reply:
mfw I originally started writing a multi-return structure and second-guessed myself that the reviewer (almost always you) would prefer a single-return. 😹

If you really feel strongly about it, I'll go back to what I was going to do in the first place.

Reply:
I usually prefer a multi-return structure when the only thing I need is to return a value, without modification shared between cases.

For instance, I'd do a `if/elif/else` if the result of this block had to be modified by something in all cases. Otherwise, multi-return has my preference.

Given that you wanted to go with that in the first place, I think it's sane to revert to it. Unless you want to have - how did we say? - room for improvement later? 😄

Reply:
I always like leaving room for improvement, but couldn't think of what might need to be shoehorned in later. So this is restructured.